plugin/pkg/admission/resourcedefaults/admission.go

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Give this package a doc.go file explaining what it does (same as your PR) since this is an "example"